glusterd2/store/config.go

```diff
@@ -16,14 +15,28 @@ import (
 )
 
 const (
-	noEmbedOpt       = "noembed"
-	etcdEndpointsOpt = "etcdendpoints"
-	etcdCURLsOpt     = "etcdcurls"
-	etcdPURLsOpt     = "etcdpurls"
-	etcdLogFileOpt   = "etcdlogfile"
-
+	// etcd client options
+	noEmbedOpt = "noembed"
+	// TODO: Remove noembed as config option, presence of "etcdendpoints" would be sufficient
```

Comment:
Not really. If "noEmbed" is set, the store will use `etcd/clientv3`. If not (by default), store will use `elasticetcd` as the etcdclient. `elasticetcd` would start its embedded etcd instance when connected to an external etcd cluster as no active leaders would exist and it would vote itself as the leader.